.env.default

```diff
@@ -2,8 +2,19 @@ export APP_ENVIRONMENT=development
 
 export DATABASE_URL=mysql://<user>:<password>@<host>/<database>
 
+export MYSQL_HOST=
+export MYSQL_USERNAME=
+export MYSQL_PASSWORD=
+export MYSQL_DATABASE=
+
 export EMAIL_TRANSPORT_DEFAULT_URL=smtp://<email>:<password>@<host>:<port>
+
 export EMAIL_FROM=
+export EMAIL_HOST=
+export EMAIL_PORT=
+export EMAIL_USERNAME=
+export EMAIL_PASSWORD=
+
```

Comment:
So what were the arguments for separate environment variables additional to or replacing the combined URL ones?